doc/contributing/release-process.rst

```diff
@@ -7,517 +7,502 @@ new CKAN release.
 
 .. seealso::
 
+   :doc:`/maintaining/releases`
+     An overview of the different kinds of CKAN release and the supported
+     versions
+
    :doc:`/maintaining/upgrading/index`
-     An overview of the different kinds of CKAN release, and the process for
-     upgrading a CKAN site to a new version.
+     The process for upgrading a CKAN site to a new version.
+
+
 
 ----------------
 Process overview
 ----------------
 
-.. versionchanged:: 2.6
-
-The process of a new release starts with the creation of a new release development branch.
+CKAN releases are always done from tags created in release development branches.
 A release development branch is the one that will be stabilized and eventually become the actual
 released version. Release branches are always named ``dev-vM.m``, after the
-:ref:`major and minor versions <releases>` they include. Major and minor versions are
-always branched from master. When the release is actually published a patch version number is added
-and the release is tagged in the form ``ckan-M.m.p``. All backports are cherry-picked on the ``dev-vM.m`` branch.
+:ref:`major and minor versions <releases>` they include, and they are always created from the ``master``
+branch. When the release is actually published a patch version number is added
+and the release is tagged in the form ``ckan-M.m.p``. All backports are cherry-picked to the
+relevant ``dev-vM.m`` branch (:ref:`automatically <automated_backports>` or manually).
 
 
  ::
 
      +--+-----------------------------------------+------------->  Master
         |                                         |
-        +-----+-------------+------>  dev-v2.6    +------->  dev-v2.7
+        +-----+-------------+------>  dev-v2.10   +------->  dev-v2.11
```

Comment:
WDYT about adding some more space between `dev-v2.10` and the `+------->  dev-v2.11`, or maybe pushing the `dev-v2.11` branch to a different line? Visually it almost looks like one dev branch merges into the next. Would be nice to make it clear that doesn't happen.

Reply:
good call, done